src/constants.ts

```diff
@@ -20,5 +20,3 @@ export const CLIENT_ID = '95109';
 export const PROTOCOL_PREFIX = 'wpcom-local-dev';
 export const WP_AUTHORIZE_ENDPOINT = 'https://public-api.wordpress.com/oauth2/authorize';
 export const SCOPES = 'global';
-export const DEFAULT_PHP_VERSION = '8.0';
```

Comment:
The default version provided by `wp-now` is also `8.0`:
https://github.com/Automattic/studio/blob/939839445bba024a7b7d5688007fbc67c57ccfcd/vendor/wp-now/src/constants.ts#L20